README.md

```diff
@@ -10,11 +10,368 @@
+  "dbPasswordFile": "test/config/db_password",
+  "fetchConfig": {
+    "interval": "20s",
+    "timeout": "5s"
+  },
+  "submitConfig": {
+    "interval": "5s",
+    "timeout": "5s",
+    "certIssuerKeyPath": "/test/issuer.key",
+    "certIssuerPath": "/test/issuer.pem"
+  },
+  "inclusionConfig": {
+    "interval": "30s",
+    "maxGetEntries": 3000
+  },
+  "logs": [
+    {
+      "uri": "http://log-one:4600",
+      "key": "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEYggOxPnPkzKBIhTacSYoIfnSL2jPugcbUKx83vFMvk5gKAz/AGe87w20riuPwEGn229hKVbEKHFB61NIqNHC3Q==",
+      "windowStart": "2000-01-01T00:00:00Z",
+      "windowEnd": "2001-01-01T00:00:00Z",
+      "maximum_merge_delay": 120,
+      "submitPreCert": false,
+      "submitCert": true
+    },
+    {
+      "uri": "http://log-two:4601",
+      "key": "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEKtnFevaXV/kB8dmhCNZHmxKVLcHX1plaAsY9LrKilhYxdmQZiu36LvAvosTsqMVqRK9a96nC8VaxAdaHUbM8EA==",
+      "windowStart": "2019-01-01T00:00:00Z",
+      "windowEnd": "2099-01-01T00:00:00Z",
+      "maximum_merge_delay": 120,
+      "submitPreCert": false,
+      "submitCert": true
+    }
+  ]
+}
+```
+
+* **metricsAddr** - a bind address for the `ct-woodpecker` Prometheus metrics
+  server.
+
+* **dbURI** - a [MySQL DSN URL](https://github.com/go-sql-driver/mysql#dsn-data-source-name) specifying
+  the DB username, address, and database name. _NOTE:_ The database user
+  password should be provided in a separate file via the "dbPasswordFile" config
+  parameter.
+
+* **dbPasswordFile** - a filepath for a file containing the DB user password.
+  _NOTE_: File must have mode `0600`.
+
+* **fetchConfig** - global configuration related to periodic STH fetching.
+
+  * **interval** - a duration string describing the time period between fetching STHs
+
+  * **timeout** - a duration string describing the timeout for fetching an STH.
+
+* **submitConfig** - global configuration related to periodic cert issuance and
+  submission. May be omitted.
+
+  * **interval** - a duration string describing the time period between attempts
+    to issue and submit certs/precerts.
+
+  * **timeout** - a duration string describing the timeout for submitting
+    a cert/precert.
+
+  * **certIssuerKeyPath** - a filepath for a file containing a PEM encoded
+  RSA/ECDSA private key corresponding to the public key in the
+  **certIssuerPath** PEM encoded intermediate certificate.
+
+  * **certIssuerPath** - a filepath for a file containing a PEM encoded x509
+  certificate to use as the issuer for certificates generated for submitting
+  to logs.
+
+* **inclusionConfig** global configuration related to checking that certificates
+  issued periodically by `ct-woodpecker` were included in the monitored logs.
+
+  * **interval** - a duration string describing the time period between attempts
+    to check unseen certificates for inclusion.
+
+  * **maxGetEntries** - the maximum number of cert entries to ask a log for.
```

Comment:
This is not quite right. Instead it should be "The maximum number of log entries to process each interval. `ct-woodpecker` will make a series of `get-entries` calls until it gets this many entries or reaches the tree head, and then process all fetched entries."